### DISCUSSION TITLE
RE7 VR PC Screen is stretched help

### DISCUSSION THREAD

**Discussion by UmutKzltg** [praydog/REFramework#465] [General Questions] [answered]
i want to stream re7 vr but the screen on my pc is very stretched i tried everything including disabling borderless window mode changing resolutions and deleting openvr but none of em fixed it help plz

**Comment by GeZorTenPlotZ** [comment]
me too. don't know what to do.

**Comment by praydog** [comment, marked as answer]
It's normal. Use the OBS OpenVR plugin to stream it, or Oculus Mirror.
